internal/resource/route.go

```diff
@@ -6,13 +6,16 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/gofrs/uuid"
```

Comment:
Merge blocker. Please use google/uuid instead in the upstream library and remove this dependency from this patch.

Reply:
done